sourcecode/apis/contentauthor/app/H5PLibrary.php

```diff
@@ -219,7 +220,7 @@ public function getAddons()
 
     public function supportsMaxScore(): bool
     {
-        $libraryLocation = sprintf('libraries/%s/presave.js', self::getLibraryString(true));
```

Comment:
It's questionable if this is an improvement, as the ContentStorageSettings class is a real rat's nest which makes any code using it hard to read. I would prefer if this were left alone, but it's up to you.